### PR TITLE
LPS-107956 Get element by tags names and classes names

### PR DESCRIPTION
**Pull request by carolmariaabb**

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js [hunk 1]
```diff
@@ -168,7 +168,7 @@ class Form extends Component {
 				this._handleAddFieldButtonClicked.bind(this)
 			),
 			dom.on(
-				`#${namespace}ControlMenu *[title="Back"]`,
+				`#${namespace}controlMenu .sites-control-group li:nth-child(2) span.lfr-portal-tooltip`,
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js]: Hey @carolmariaabb, is the property `nth-child` mandatory to address this issue? I'm worried about the addition of a new item to the menu and the Back button change its position from 2 to other, causing another rework from our side.

Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js]: Hey @natocesarrego, I agree with you. I removed the `li:nth-child(2)` property and it still works. Thanks!

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
Hi @natocesarrego, I executed `DDMFormInstanceRecordSearchTest` and `DDMStructureLocalServiceTest` tests on my local master branch and they both failed.

**Comment by natocesarrego** [comment]
Thanks for the confirmation, @carolmariaabb. 😉

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/84654. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**